external/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/OracleIntegrationSuite.scala

```diff
@@ -62,6 +62,12 @@ class OracleIntegrationSuite extends DockerJDBCIntegrationSuite with SharedSQLCo
   }
 
   override def dataPreparation(conn: Connection): Unit = {
+    conn.prepareStatement("CREATE TABLE datetime (id NUMBER(10), d DATE, t TIMESTAMP)")
+      .executeUpdate()
+    conn.prepareStatement("INSERT INTO datetime VALUES ("
+      + "1, {d '1991-11-09'}, {ts '1996-01-01 01:23:45'})").executeUpdate()
+    conn.prepareStatement("CREATE TABLE datetime1 (id NUMBER(10), d DATE, t TIMESTAMP)")
```

Comment:
Thank you for reviewing the patch. I think cleanup is not required, these tables are not persistent across the test runs. They are  cleaned up when docker container is removed at the end of the test.  Currently I did notice any setup in the afterAll() to do it after the test.

I moved up creation of temporary views also  to the same place, to keep them together. And possibly any future  tests can also use these tables.